packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -408,3 +421,48 @@ const generateMissingOrAliasedRequiredFields = ({
 
     return generateMissingOrAliasedFields({ fieldNames: requiredFields, selection });
 };
+
+function createFulltextProjection({
+    resolveTree,
+    node,
+    context,
+    chainStr,
+    varName,
+    literalElements,
+    resolveType,
+}: {
+    resolveTree: ResolveTree;
+    node: Node;
+    context: Context;
+    chainStr?: string;
+    varName: string;
+    literalElements?: boolean;
+    resolveType?: boolean;
+}): ProjectionResult {
+    if (!resolveTree.fieldsByTypeName[node.fulltextTypeNames.result][lowerFirst(node.name)]) {
+        return {
+            projection: "{ }",
+            params: {},
+            meta: {},
+            subqueries: [],
+            subqueriesBeforeSort: [],
+        };
+    }
+
+    const nodeResolveTree = {
+        ...resolveTree.fieldsByTypeName[node.fulltextTypeNames.result][lowerFirst(node.name)],
+    };
```

Comment:
Do we need to clone this? Is it supposed to be modified in the method `createProjectionAndParams`?